tests/test_mathematical.py

```diff
@@ -137,16 +138,7 @@ def _test_mathematical(self, name, dtype, lhs, rhs):
         else:
             result = getattr(dpnp, name)(a_dpnp, b_dpnp)
             expected = getattr(numpy, name)(a_np, b_np)
-            if numpy.issubdtype(expected.dtype, numpy.floating):
-                tol = numpy.max(
-                    [
-                        numpy.finfo(result.dtype).resolution,
-                        numpy.finfo(expected.dtype).resolution,
-                    ]
-                )
-            else:
-                tol = 1e-06
-            assert_allclose(result, expected, rtol=tol)
+            assert_dtype_allclose(result, expected)
```

Comment:
looks we need to extend `assert_dtype_allclose` with new keyword like `check_type` since part of the tests are failing now:
```
FAILED tests/test_mathematical.py::TestMathematical::test_fmod[int32-lhs0-3] - AssertionError
FAILED tests/test_mathematical.py::TestMathematical::test_fmod[int32-lhs1-3] - AssertionError
FAILED tests/test_mathematical.py::TestMathematical::test_maximum[int32-lhs0-3] - AssertionError
FAILED tests/test_mathematical.py::TestMathematical::test_maximum[int32-lhs1-3] - AssertionError
FAILED tests/test_mathematical.py::TestMathematical::test_minimum[int32-lhs0-3] - AssertionError
FAILED tests/test_mathematical.py::TestMathematical::test_minimum[int32-lhs1-3] - AssertionError
```